presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoResultSet.java

```diff
@@ -1700,6 +1704,11 @@ private int columnIndex(String label)
         return index;
     }
 
+    private void setWasNull(Object value)
+    {
+        wasNull.set(value == null ? true : false);
```

Comment:
The expression `value == null` is already a boolean, so the `? true : false` is redundant (IntelliJ's "Redundant conditional expression" inspection will flag this). So we can simplify that, then inline the method.